github/GithubObject.py

```diff
@@ -151,6 +151,10 @@ def _makeStringAttribute(value):
     def _makeIntAttribute(value):
         return GithubObject.__makeSimpleAttribute(value, (int, long))
 
+    @staticmethod
```

Comment:
I added this because the add_attribute script wants it to exist - but I'm not sure any API's have floating point attributes, so might be better to remove this method and remove support for `float` from add_attribute.

Reply:
Hmm, don't think Github will have any `float` attributes, so remove it from the `add_attribute` script altogether?